satnogsclient/scheduler/tasks.py

```diff
@@ -30,9 +30,19 @@ def spawn_observation(*args, **kwargs):
 
 def get_jobs():
     """Query SatNOGS Network API to GET jobs."""
+
+    print 'Try getting Jobs from Network ..'
+
     url = urljoin(settings.NETWORK_API_URL, 'jobs')
     params = {'ground_station': settings.GROUND_STATION_ID}
-    response = requests.get(url, params=params)
+    try:
+        response = requests.get(url, params=params, verify=settings.CA_CERT)
+    except:
+        print 'HTTPs certificate verification failed..(ignore for now)'
+        response = requests.get(url, params=params, verify=False)
```

Comment:
I agree, but that's what the settings variable is for. If `True` it means we really don't want to overlook the invalid certificate.

I would do something like that:

```
from requests.exceptions import SSLError

try:
    response = requests.get(url, params=params, verify=settings.VERIFY_SSL)
except SSLError:
    print 'HTTPs certificate verification failed'
    sys.exit(0)
except:
    print 'Connections failed'
    sys.exit(0)
```
So if `VERIFY_SSL=False` we'll never get an SSLError exception and we overcome the current problem. But the same code with `VERIFY_SSL=False` can be used in production too, where we probably want to avoid security issues like MITM attacks.